src/Pate/SimState.hs

```diff
@@ -162,6 +195,36 @@ instance OrdF (W4.SymExpr sym) => Semigroup (AssumptionSet sym v) where
     binds = mergeExprSetMap (Proxy @sym) (asmBinds asm1) (asmBinds asm2)
     in AssumptionSet preds binds
 
+ppExprSet ::
+  W4.IsExpr (W4.SymExpr sym) =>
+  Proxy sym ->
+  ExprSet sym tp ->
+  PP.Doc a
+ppExprSet _ es =
+  let ps = [ W4.printSymExpr p | p <- SetF.toList es ]
+  in PP.sep (zipWith (<+>) ("{" : repeat ",") ps) <+> "}"
```

Comment:
Any reason this isn't in `ExprHelpers` with the definition of `ExprSet`?

Reply:
Moved